std_msgs/Header header
bool success
geometry_msgs/PoseArray[] plans